apps/gateway-ui/src/components/DepositCard.tsx

```diff
@@ -44,95 +44,81 @@ export const DepositCard = React.memo(function DepositCard({
                 fontFamily={theme.fonts.body}
-                overflowWrap='break-word'
-                wordBreak='break-word'
-                cursor='pointer'
                 onClick={onCopy}
+                p='4px 12px'
+                borderRadius='16px'
+                border={`1px solid ${theme.colors.gray[400]}`}
               >
-                {address}
+                {t('common.copied')}
               </Text>
             ) : (
-              <Text
-                fontSize='md'
-                color={theme.colors.red}
-                fontFamily={theme.fonts.body}
-              >
-                {error}
-              </Text>
-            )}
-            <Box>
-              {hasCopied ? (
-                <Text
-                  fontSize='xs'
-                  fontWeight='semibold'
-                  color={theme.colors.gray[900]}
-                  bgColor={theme.colors.gray[25]}
-                  fontFamily={theme.fonts.body}
-                  onClick={onCopy}
-                  p='4px 12px'
-                  borderRadius='16px'
-                  border={`1px solid ${theme.colors.gray[400]}`}
-                >
-                  {t('common.copied')}
-                </Text>
-              ) : (
-                <CopyIcon
-                  color={theme.colors.gray[900]}
-                  height='20px'
-                  width='20px'
-                  onClick={onCopy}
-                  cursor='pointer'
-                />
-              )}
-            </Box>
-          </Flex>
-          <Box>
-            {address && (
-              <QRCodeSVG height='200px' width='200px' value={address} />
+              <CopyIcon
+                color={theme.colors.gray[900]}
+                height='20px'
+                width='20px'
+                onClick={onCopy}
+                cursor='pointer'
+              />
             )}
           </Box>
-        </Stack>
-        <Flex gap='4px'>
-          <Link
-            fontSize='md'
-            color={theme.colors.blue[600]}
-            fontFamily={theme.fonts.body}
-            _hover={{ textDecoration: 'underline' }}
-            transition={`text-decoration 1s ease-in-out`}
-            href={`https://mempool.space/address/${address}`}
-            target='_blank'
-            rel='noreferrer'
-            w='fit-content'
-          >
-            {t('deposit-card.mempool_deposit_link_text')}
-          </Link>
-          <LinkIcon color={theme.colors.blue[600]} />
         </Flex>
+        <Box>
+          {address && (
+            <QRCodeSVG height='200px' width='200px' value={address} />
+          )}
+        </Box>
       </Stack>
+      <Flex gap='4px'>
+        <Link
+          fontSize='md'
+          color={theme.colors.blue[600]}
+          fontFamily={theme.fonts.body}
+          _hover={{ textDecoration: 'underline' }}
+          transition={`text-decoration 1s ease-in-out`}
+          href={`https://mempool.space/address/${address}`}
```

Comment:
Random thought: Settings to configure what mempool you want to use or LN explorer, etc. Def future fedimint problems to address.

Reply:
Agreed. It's _slightly_ less of a concern since we're not using these as APIs, just linking out, but configurability is king.

Reply:
#181